test/docker.js

```diff
@@ -433,6 +433,7 @@ describe('images', function () {
         expect(images).to.have.length(2);
         expect(images[0].RepoTags).to.have.length(1);
         expect(images[0].RepoTags[0]).to.equal('testImage:latest');
+        expect(images[0].Created).to.be.a.number();
```

Comment:
you can use `.to.be.about(new Date().valueOf() / 1000 | 0, 10);` here (which, from my quick understanding, would be "within ten seconds")

https://github.com/hapijs/code#aboutvalue-delta